data/types.go

```diff
@@ -90,42 +90,52 @@ type Role struct {
 
 type Files map[string]FileMeta
 
+type FileMeta struct {
+	Length int64            `json:"length",omitempty`
+	Hashes Hashes           `json:"hashes",ompitempty`
```

Comment:
```suggestion
	Hashes Hashes           `json:"hashes",omitempty`
```
I don't speak Go, but this looks like a typo?